scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/remotes/postgres/GetRequestAction.java

```diff
@@ -12,11 +10,14 @@
 /*package-local*/ final class GetRequestAction implements AutoCloseable {
   private final @Language("SQL") String sql = """
     select
-      req.analysis_id
-    from scheduling_request as req
-      where
-        req.specification_id = ? and
-        req.specification_revision = ?
+      r.analysis_id,
+      r.status,
+      r.failure_reason,
+      r.canceled
+    from scheduling_request as r
+    where
+      r.specification_id = ? and
```

Comment:
Oop! We should make a call once and for all between `specification` and `spec`. This column name does not exist.

Reply:
I'm going to probably go with specification for all except low-scope things like local variables, or cases of `as spec` for database tables in queries as above. I'll update you here once I've addressed this.

Reply:
Alright, I added a commit to the start of this branch changing all instances of `spec` to `specification` in the database files. I stopped at every commit along the way in that rebase to make sure all postgres actions reflected this change.

Reply:
I'll take the spaceship as a sign this is good to go, especially since the issue has been addressed, and Matt reached out to me to ask when I'm going to merge.